DuckDuckGo/TabViewController.swift

```diff
@@ -1153,6 +1160,14 @@ extension TabViewController: WKNavigationDelegate {
             return true
         }
         
+        if AppDependencyProvider.shared.variantManager.isSupported(feature: .parameterStripping),
+           let newUrl = linkCleaner.cleanTrackingParameters(initiator: webView.url,
+                                                            url: navigationAction.request.url) {
+            decisionHandler(.cancel)
```

Comment:
Maybe we should double-check that new url is actually different from the old one before cancelling?